### PR TITLE
Readme updated 

### PR DESCRIPTION
**Pull request by MashB**
Readme updated based on review feedback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,14 +62,14 @@ Application secrets are not included in the code repository. Below are the instr
...
-Create **developer-local-properties.yaml** file under root of `resource` folder and set the application.yaml
+Create **developer-local-properties.yaml** file under root of `resource` folder and override the application.yaml
 placeholders.
 
 ```src/main/resources/developer-local-properties.yaml```
 
 ###### PROD
 
-Secrets will be set as an environment variables on the deployment environment.
+Secrets are configured as an environment variables on the deployment server.
```
Comment (susrisha) [inline README.md:72]: Remove `an` . The sentence should be
Secrets are configured as environment variables on the deployment server.

**Review by susrisha — CHANGES_REQUESTED** [review]
Two minor changes

File: README.md [hunk 2]
```diff
@@ -17,14 +18,16 @@ graph LR;
...
-  api-key and generates the secret token
+- `Auth Service` is responsible for Authenticating and Authorizing the HTTP requests. Also, verifies the access token,
+  api-key on request and generates the secret token
   for intra micro-service communication.
-- Gateway service is first point of contact for clients. Every request to gateway is Authenticated & Authorized against
-  the Auth service.
-- For Authentication, Auth service will interact with keycloak service for verification.
-    - Successful authentication auth servive will return with access_token & refresh_token.
-- For Authorization, Auth service will interact directly with TDEI database.
+- `Gateway` service is first point of contact for clients. Every request to gateway is Authenticated & Authorized
```
Comment (rdevalap) [inline README.md:24]: `Gateway` service is **the** first point of contact

File: README.md [hunk 3]
```diff
@@ -17,14 +18,16 @@ graph LR;
...
+  against
+  the `Auth Service`.
+- For Authentication, `Auth Service` will interact with `Keycloak Service` for verification.
```
Comment (rdevalap) [inline README.md:27]: For Authentication, `Auth Service` interacts with `Keycloak` for verification. Please write it in "present tense".

File: README.md [hunk 4]
```diff
@@ -17,14 +18,16 @@ graph LR;
...
+    - Successful authentication auth service will return with access_token & refresh_token as a response to requests.
```
Comment (rdevalap) [inline README.md:28]: Same here, 
On successful authentication, auth service returns _access_token_ & _refresh_token_ as response

File: README.md [hunk 5]
```diff
@@ -17,14 +18,16 @@ graph LR;
...
+- For Authorization, `Auth Service` will interact directly with `TDEI Database`. `TDEI Database` maintains the user
```
Comment (rdevalap) [inline README.md:29]: For Authorization, Service interacts directly with `TDEI Database`

**Review by rdevalap — COMMENTED** [review]
Is the URL at the bottom correct?
https://tdei-gateway.azurewebsites.net/swagger-ui/index.html?

**Review by rdevalap — APPROVED** [review]

**Review by susrisha — APPROVED** [review]

**Pull request merged**